app/controllers/emails_controller.rb

```diff
@@ -23,6 +24,11 @@ def submit
     end
   end
 
+  def submit_generic
+    @send_generic = true
+    @templates = []
+    send_generic
+  end
```

Comment:
There is no enabled draft sending for thoose emails, as the drafts always refer to an acceptance or rejection email, which is not the case here (and if I'm honest, I was too lazy to also adapt the template part of the mail system). But on the other hand I could disable the functionality in the view then. Mh.